mesh2hrtf/Mesh2Input/mesh2input.py

```diff
@@ -83,6 +85,13 @@ class ExportMesh2HRTF(bpy.types.Operator, ExportHelper):
         description="Render pictures of the 3D mesh",
         default=True,
         )
+    
+    checkall: BoolProperty(
+        name="3D-Print Checks",
+        description= ("Check geometry"),   #Added by Sanket
+        default=True,
+    )
```

Comment:
- A general question: should we have separate checks or only one box that always does all checks?
- Lets find a different name for `checkall` this could be anything
- Lets find a different name for '3D-Print checks' since Mesh2HRTF is not about printing
- The description should be more specific: What is checked for which mesh?
- There should not be any `#Added by Sanket` or similar comments. Git keeps track of who added which code...